process/coordinator/process.go

```diff
@@ -708,8 +708,9 @@ func (tc *transactionCoordinator) CreateMarshalizedData(body *block.Body) map[st
 			continue
 		}
 
+		isPreProcessMiniBlock := miniBlock.Type == block.TxBlock
```

Comment:
only txBlock miniblocks can be "preprocessed" ?

Reply:
marshalled data for cross shard can come from preprocessor only in case of transactions.

Reply:
ok